### PR TITLE
Adding iOS guide

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

File: src/pages/guides/ios.mdx [hunk 1]
```diff
@@ -0,0 +1,220 @@
+---
+title: Using Web3Auth's iOS SDK
```
Comment (yashovardhan) [inline src/pages/guides/ios.mdx:2]: Title should be `Using Web3Auth iOS SDK`

File: src/pages/guides/ios.mdx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+This guide will cover the basics of how to use the Web3Auth iOS SDK in your iOS application.
+
+Repository: https://github.com/Web3Auth/examples/raw/main/ios
+
+## Quick Start
+
+```bash
+git clone https://github.com/Web3Auth/examples.git
+cd ios
```
Comment (yashovardhan) [inline src/pages/guides/ios.mdx:32]: cd `ios-example`?

Comment (yashovardhan) [reply src/pages/guides/ios.mdx:32]: https://github.com/Web3Auth/examples/tree/main/ios/ios-example

File: src/pages/guides/ios.mdx [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+- When a user logs in with `Google`, Google sends a JWT `id_token` to the app. This JWT token is sent to the Web3Auth SDK's login function.
+
+- Finally, on successful validation of the JWT token, Web3Auth SDK will generate a private key for the user, in a self custodial way, resulting in
+  easy onboarding for your user to the application.
+
+## Prerequisites
+
+---
+
+<Web3AuthPrerequisites />
```
Comment (yashovardhan) [inline src/pages/guides/ios.mdx:53]: Pre requisites need to be updated for this.

Comment (yashovardhan) [reply src/pages/guides/ios.mdx:53]: Add iOS 14 and above

File: src/pages/guides/ios.mdx [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+<dict>
+  <key>ClientId</key>
+  <string>YOUR_OPENLOGIN_CLIENT_ID</string>
+  <key>Network</key>
+  <string>mainnet | testnet</string>
+</dict>
+</plist>
+```
+
+When you have a Web3Auth.plist and you have specified your Web3Auth configuration there, you can simply construct your Web3Auth instance with
```
Comment (yashovardhan) [inline src/pages/guides/ios.mdx:116]: Add `` to Web3Auth.plist

File: src/pages/guides/ios.mdx [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+
+```swift
+import Web3Auth
+
+let web3auth = Web3Auth()
+```
+
+#### Without `Web3Auth.plist`
+
+If you don't have a Web3Auth.plist or want to use optional configurtion parameters, you can construct the Web3Auth instance with
```
Comment (yashovardhan) [inline src/pages/guides/ios.mdx:126]: Add `` to Web3Auth.plist

File: src/pages/guides/ios.mdx [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+let web3auth = Web3Auth(W3AInitParams(
+  clientId: "<your clientId>",
+  network: .MAINNET,
+  sdkUrl: ...,
+  redirectUrl: ...,
+))
+```
+
+All the below code snippets are based on the above initialization.
+
```
Comment (yashovardhan) [inline src/pages/guides/ios.mdx:140]: Add link to SDK Reference that for more information check the SDK reference for initialisation section

**Review by yashovardhan — COMMENTED** [review]
Overall looks good, do these small changes and ping me to merge asap

**Comment by Saviour1001** [comment]
Got the changes done !

**Review by shahbaz17 — APPROVED** [review]
LGTM

**Pull request merged**